src/status_im/components/toolbar_new/styles.cljs

```diff
@@ -90,8 +90,8 @@
    :padding-horizontal 16})
 
 (defstyle item
-  {:ios {:margin-horizontal 12
-         :margin-vertical   16}
+  {:ios {:marginHorizontal 12
```

Comment:
could you elaborate? and map indentation is wrong

Reply:
I have to double check this. For some reason I had warnings in iOS simulator about unknown properties because they were not properly translated.